spec/index.md

```diff
@@ -5078,6 +5082,61 @@ S with
 
 ```
 
+#### IC Management Canister: List snapshots
```

Comment:
Take snapshot is missing as an IC mgmt canister method here in the formal semantics